plat/common/include/pci/pci_bus.h

```diff
@@ -226,6 +226,10 @@ static struct pci_bus_handler ph __unused;
 #define PCI_CONF_SUBCLASS_ID_SHFT     (16)
 #define PCI_CONF_SUBCLASS_ID_MASK     (0x00FF)
 
+#define PCI_CONF_COMMAND              (0x04)
```

Comment:
This goes in a separate commit, e.g. `plat/common/pci: Add PCI_CONF_COMMAND macro`